lib/exq/enqueue_api.ex

```diff
@@ -98,6 +98,20 @@ defmodule Exq.Enqueuer.EnqueueApi do
         queue_adapter = Config.get(:queue_adapter)
         queue_adapter.enqueue_in(pid, queue, offset, worker, args, options)
       end
+
+      @doc """
+      Schedule multiple jobs to be atomically enqueued at specific times
+
+      Expected args:
+      * `pid` - PID for Exq Manager or Enqueuer to handle this
+      * `jobs` - List of jobs
```

Comment:
could you include sample jobs spec (like the one in the README) here as well

Reply:
sure

Reply:
<img width="707" alt="Screenshot 2023-01-23 at 11 43 44 PM" src="https://user-images.githubusercontent.com/12698844/214214626-2cc426d5-19c0-4143-bf22-dcca079e3e06.png">